charts/nginx-ingress/values.yaml

```diff
@@ -173,7 +173,8 @@ controller:
       type: RuntimeDefault
 
   ## The security context for the Ingress Controller containers.
-  securityContext: {} # Remove curly brackets before adding values
+  securityContext:
```

Comment:
is this a linter issue

Reply:
object type so leaving it empty throws an error:
`Invalid type. Expected: object, given: null`

Reply:
Must have been a mistake as linter seems fine, I've reverted